src/devices/sound/mas3507d.cpp

```diff
@@ -59,6 +53,7 @@ mas3507d_device::mas3507d_device(const machine_config &mconfig, const char *tag,
 void mas3507d_device::device_start()
 {
 	stream = stream_alloc(0, 2, 44100);
+	mp3dec = std::make_unique<mp3_audio>(static_cast<const uint8_t *>(&mp3data[0]));
```

Comment:
This should really be a `reinterpret_cast` if you’re trying to preserve the pointer value.